### PR TITLE
Throw event each time a currency network gets add

### PR DESCRIPTION
**Pull request by weilbith**
Closes: trustlines-network/project#646

### CODE REVIEW

File: tests/test_currency_network_registry.py [hunk 1]
```diff
@@ -92,6 +92,66 @@ def test_add_network_multiple(
     )
 
 
+def test_add_network_twice_does_not_change_metadata(
+    initialized_currency_network_registry_contract, currency_network_contract, accounts
+):
+    """Adding the same network again does not change the initial information.
+
+    Each new registration does throw an event with the current
+    registrar, but the stored metadata from the first call do not change
```
Comment (cducrest) [inline tests/test_currency_network_registry.py]: Each new registration emits an event with the new registrar, but the original registrar from the first call does not change.

File: tests/test_currency_network_registry.py [hunk 2]
```diff
@@ -92,6 +92,66 @@ def test_add_network_multiple(
...
+        == currency_network_metadata[0]
+    )
+
+
+def test_add_network_multiple_events(
+    currency_network_registry_contract, currency_network_contract, accounts
+):
+    """Adding the same network with multiple accounts.
+
+    Each call throws a new event with the according account as
```
Comment (cducrest) [inline tests/test_currency_network_registry.py]: Each call emits a new event with the corresponding ...

File: tests/test_currency_network_registry.py (hunk 1 above)
Comment (weilbith) [reply tests/test_currency_network_registry.py]: :heavy_check_mark:

File: tests/test_currency_network_registry.py (hunk 2 above)
Comment (weilbith) [reply tests/test_currency_network_registry.py]: :heavy_check_mark:

**Pull request merged**